new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -534,7 +544,93 @@ public static boolean deletePvc(String pvcName, String namespace) throws ApiExce
 
     return false;
   }
-  // --------------------------
+
+  // --------------------------- service account ---------------------------
+
+  /**
+   * Create a service account for a given namespace
```

Comment:
tiny nitpick - the javadoc suggest that we are providing a namespace as a param, but it is actually embedded in the SA object, i would remove the "for a given namespace" just to avoid confusion